modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_mixins.scss

```diff
@@ -1,27 +1,42 @@
-@mixin button-cp(
+@mixin custom-properties-button(
```

Comment:
I think we could do away with this mixin and use the `clay-button-variant` mixin. It has a lot of features. If we want to be able to customize more in the future we can just add a property and the mixin will output it. It would look something like this:
```
$custom-properties-btn-primary: () !default;
$custom-properties-btn-primary: map-deep-merge(
	(
		background-color: var(--primary),
		border-color: transparent,
		color: var(--white),
		hover: (
			background-color: var(--primary),
			border-color: transparent,
			color: var(--white),
		),
		focus: (
			background-color: var(--primary),
			border-color: transparent,
			color: var(--white),
		),
		active: (
			background-color: var(--primary),
			border-color: transparent,
			color: var(--white),
		),
	),
	$custom-properties-btn-primary
);

.btn-primary {
	@include clay-button-variant($custom-properties-btn-primary);
}
```